docs/fsharp/language-reference/discriminated-unions.md

```diff
@@ -81,6 +81,25 @@ let getShapeHeight shape =
 
 Normally, the case identifiers can be used without qualifying them with the name of the union. If you want the name to always be qualified with the name of the union, you can apply the [RequireQualifiedAccess](https://msdn.microsoft.com/library/8b9b6ade-0471-4413-ac5d-638cd0de5f15) attribute to the union type definition.
 
+### Unwrapping Discriminated Unions
+
+In F# Discriminated Unions are often used in Domain-Modelling for wrapping a single type.
+To ease usage of such single-case discriminated unions, F# provides a nice handy syntax to unwrap them fast.
```

Comment:
"To ease usage of such single-case discriminated unions, F# provides a nice handy syntax to unwrap them fast."

I'd change that to this:

"It's easy to extract the underlying value via Pattern Matching as well.  You don't need to use a `match` espression for a single case:"